.claude/knowledge/ciso/decisions.md

```
@@ -102,3 +102,89 @@ WORKTREE ⊇ HOME 注入（`WORKTREE=/Users` / `$HOME/subdir` / symlink 経由
+activate.sh 内の `bin_abs="$(cd "$(dirname "$script_path")" && pwd)"` は source 実行時に物理パスを解決する。ここはインストール後のユーザー実行フェーズであり、install.sh の配置フェーズとは別問題。REPO_ROOT の symlink 解決は `detect_repo_root` 関数で行うべきだが、これは既存設計の課題でありPhase 3a 新規変更の直接的な脆弱性ではない。
+
+**判定: High → Low（Phase 2 以降トラッキング）**。activate.sh の PATH 注入は `pwd` 経由で物理パスを取得しており、symlink 解決は呼び出し元シェルの PATH 展開の問題。設計変更が必要なレベルであり Phase 3a のスコープを超える。
+
+### 採用する指摘
+
+**Minor — Phase 3a で修正すべき**
+
+| ID | 検出者 | 内容 | 採用根拠 |
+|----|--------|------|----------|
+| P318-001 | 全員一致（3/3） | `copy_isolation_templates` の `[[ -f "$src" ]]` がシンボリックリンクを通過する | 3名全員一致。`[[ ! -L "$src" ]]` の追加で排除可能 |
+| P318-003 | Analyst 3 単独 | `cleanup()` の `cd "$SCRIPT_DIR"` に `|| true` なし。`.claude/rules/testing.md` 規約違反 | 規約明記の Trivial。単独指摘だが testing.md に明記された MUST 事項のため採用 |
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)